Formula/kubie.rb

```diff
@@ -0,0 +1,44 @@
+class Kubie < Formula
+  desc "Much more powerful alternative to kubectx and kubens"
+  homepage "https://blog.sbstp.ca/introducing-kubie/"
+  url "https://github.com/sbstp/kubie/archive/v0.9.1.tar.gz"
+  sha256 "8f7926b1bcd0421247231353b6969c36498add3b88b55103349df3da763a5f42"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+    bash_completion.install "./completion/kubie.bash"
+  end
+
+  test do
+    kube_config_file_path = File.join(ENV["HOME"], ".kube", "kubie-test.yaml")
+    mkdir_p(File.dirname(kube_config_file_path))
```

Comment:
```suggestion
    mkdir_p ENV["HOME"]"/.kube"
```

Reply:
This doesn't work:

```irb(main):001:0> ENV["HOME"]"/.kube"
Traceback (most recent call last):
        3: from /Users/cassiano/.rubies/ruby-2.6.5/bin/irb:23:in `<main>'
        2: from /Users/cassiano/.rubies/ruby-2.6.5/bin/irb:23:in `load'
        1: from /Users/cassiano/.rubies/ruby-2.6.5/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
SyntaxError ((irb):1: syntax error, unexpected tSTRING_BEG, expecting end-of-input)
ENV["HOME"]"/.kube"
           ^
```

You original suggestion, with the `/` between the 2 strings doesn't work either:

```irb(main):002:0> ENV["HOME"]/".kube"
Traceback (most recent call last):
        5: from /Users/cassiano/.rubies/ruby-2.6.5/bin/irb:23:in `<main>'
        4: from /Users/cassiano/.rubies/ruby-2.6.5/bin/irb:23:in `load'
        3: from /Users/cassiano/.rubies/ruby-2.6.5/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        2: from (irb):2
        1: from (irb):2:in `rescue in irb_binding'
NoMethodError (undefined method `/' for "/Users/cassiano":String)
```

Reply:
Does this work:
```
    mkdir_p "#{ENV["HOME"]}/.kube"
```
Reviewing in a web UI is kinda difficult without syntax highlighting.

Reply:
Yes, that works. 

    "#{ENV["HOME"]}/.kube/kubie-test.yaml".write "foo"

does't though, since there's no `String#write` method. It needs a `File` object.

Reply:
Okay, can you set `ENV["home"]` to `testpath` and write these files to `testpath"/stuff"` instead? I think that would be a cleaner approach to this than writing to an actual user directory

Reply:
Yes, that works - thanks!

To be clear, `ENV["HOME"]` in the context of the test is the same directory as `testpath`. I checked that it wasn't writing to the user's home before I submitted the first version of this PR.

In any case, `testpath` _is_ a `File` (or equivalent duck) object, so this is better.